metal/resource_metal_reserved_ip_block_test.go

```diff
@@ -35,7 +35,7 @@ resource "metal_project" "foobar" {
 
 resource "metal_reserved_ip_block" "test" {
 	project_id  = metal_project.foobar.id
-	facility    = "ny5"
+	facility    = "sv"
```

Comment:
This should be `metro =` (and line 104 should be metro too). Similar changes appear elsewhere.